### DISCUSSION TITLE
Welcome to Pool Discussions!

### DISCUSSION THREAD

**Discussion by prateekmedia** [prateekmedia/appimagepool#80] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anuragsingh6** [comment]
Hi there! I want to add Hindi translation. Where and how can I do this?

**Reply by prateekmedia** [reply]
Hello! Please check this: https://github.com/prateekmedia/appimagepool#contribute-translations

**Reply by anuragsingh6** [reply]
Thanks, I've added the translations please check and merge PR.
